gradle.properties

```diff
@@ -5,16 +5,15 @@ group=org.jetbrains.kotlinx
 version=0.4.1
 versionSuffix=SNAPSHOT
 
-defaultKotlinVersion=1.8.21
+defaultKotlinVersion=1.9.20
 dokkaVersion=1.8.20
-serializationVersion=1.5.1
+serializationVersion=1.6.1-wasm1
```

Comment:
I don't think it's ok to expose this version transitively

Reply:
Please comment what do you mean by expose transitively

Reply:
I replace this dependency with 1.6.1 (no wasm) but there need to be new version for kotlin 1.9.1 as far as I understand.

Reply:
@sandwwraith Could you please release kotlinx-serialization with wasm-js and 1.9.21 to maven central?

Reply:
Do you need 1.9.21 specifically? 1.6.1 already has wasm-js and is based on 1.9.20

Reply:
If 1.6.1 has linuxArm32Hfp and have not to be updated in datetime then I am perfectly fine with 1.6.1. @ilya-g

Reply:
Super! Thanks!